### DISCUSSION TITLE
Move schema.prisma and schema.graphql into `.keystone/` folder

### DISCUSSION THREAD

**Discussion by gautamsi** [keystonejs/keystone#5676] [Feature requests]
I see it was in keystone folder earlier, can we move it back there when generated?

it clutters the root folder unnecessarily

**Comment by bladey** [comment]
Thanks for the suggestion @gautamsi.

**Comment by dcousens** [comment]
You can now demark where these are using configurable output paths.
[Please see our custom output path example](https://github.com/keystonejs/keystone/tree/main/examples/custom-output-paths).
